misc/scripts/upgrade_test_vib.txt

```diff
@@ -0,0 +1,2 @@
+0.14 https://bintray.com/vmware/vDVS/download_file?file_path=VMWare_bootbank_esx-vmdkops-service_0.14.0577889-0.0.1.vib
```

Comment:
better to have some short of summary about objective of this file.... or add some note at `get_vib_url() {` where to add new entry .